opentelemetry-proto/pyproject.toml

```diff
@@ -25,7 +25,7 @@ classifiers = [
   "Programming Language :: Python :: 3.11",
 ]
 dependencies = [
-  "protobuf~=3.13",
+  "protobuf~=3.19",
```

Comment:
With this change, we should be forward compatible into the protobuf 4.x range as well!

```suggestion
  "protobuf>=3.19",
```